atsign-foundation/dess
# Use org level templates

- What I did

Deleted repo templates so that org templates will be used instead

- How to verify it

Merge and see that the pub.dev checklist is now available as an issue template

- Description for the changelog

Use org level templates
